src/cascadia/TerminalControl/Resources/en-US/Resources.resw

```diff
@@ -200,4 +200,10 @@ Please either install the missing font or choose another one.</value>
   <data name="TermControlReadOnly" xml:space="preserve">
     <value>Read-only mode is enabled.</value>
   </data>
+  <data name="SearchBox_MatchesAvailable" xml:space="preserve">
+    <value>Results found</value>

```

Comment:
should we add comments explaining that these are read aloud to users using screen readers?